backend/migrations/env.py

```
@@ -0,0 +1,113 @@
+def run_migrations_online():
+    """Run migrations in 'online' mode.
+
+    In this scenario we need to create an Engine
+    and associate a connection with the context.
+
+    """
+
+    # this callback is used to prevent an auto-migration from being generated
+    # when there are no changes to the schema
+    # reference: http://alembic.zzzcomputing.com/en/latest/cookbook.html
+    def process_revision_directives(context, revision, directives):
```

Ruff (0.11.9)

Missing return type annotation for private function `process_revision_directives`

Add return type annotation: `None`

(ANN202)